test/test_pandas.py

```diff
@@ -1,13 +1,18 @@
 import pytest
 
 
-@pytest.mark.skip
-def test_pandas(selenium):
+def test_pandas(selenium, request):
+    if 'chrome' in selenium.__class__.__name__.lower():
+        request.applymarker(pytest.mark.xfail(
+            run=False, reason='chrome not supported'))
     selenium.load_package("pandas")
     assert len(selenium.run("import pandas\ndir(pandas)")) == 179
 
 
-@pytest.mark.skip
-def test_extra_import(selenium):
+def test_extra_import(selenium, request):
+    if 'chrome' in selenium.__class__.__name__.lower():
```

Comment:
This would be needed quite often in tests, I'm not sure if there is better solution than  monkeypatching `selenium.webdriver.{Firefox,Chrome}` with the `name` attribute or similar in `tests/conftest.py`, eg..
```py
class ChromeWrapper(SeleniumWrapper):
    def get_driver(self):
        [...]
        driver = Chrome(chrome_options=options)
        driver.name = 'chrome'
        return driver
```
?

Reply:
Since they are in pyodide, I think `ChromeWrapper` and `FirefoxWrapper` could both just add a name attribute we can check against, e.g.:

```
class ChromeWrapper(SeleniumWrapper):
    name = 'chrome'
```

and then:

```
if selenium.name == 'chrome':
```

Maybe `browser` is a better name than `name`...